roles/etcd/tasks/set_facts.yml

```diff
@@ -3,3 +3,4 @@
   set_fact:
     etcd_ip: "{{ etcd_ip }}"
     etcd_hostname: "{{ etcd_hostname }}"
+    etcdctlv2: "{{ etcdctlv2 }}"
```

Comment:
I don't think we should alter set_facts as it's called elsewhere and I'm concerned it might have knock-on effects when we migrate to static pods.  Probably in 3.11 it would be safe to do this.  For 3.10, let's make this it's own task_file and specifically include it via import_role during scaleup play.